src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -222,8 +222,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   def createWorkflow(inputs: HostInputs): Future[Unit] = {
     val symbolStoreEntries = buildSymbolStoreEntries(workflow.namespace, inputs)
     symbolCache = symbolStoreEntries.groupBy(entry => SymbolCacheKey(entry.scope, entry.isInput))
+    // Currently assumes there is at most one possible final call, a `CopyWorkflowOutputs`.
+    val finalCall = workflow.workflowOutputsPath.toOption map { _ => CopyWorkflowOutputs(workflow) }
```

Comment:
This is a consequence of how `get` is implemented on `WorkflowOptions` as returning a `Try[String]`.  The only possible errors are the value is not present (which is not an error for this use case) or the value is present but is not of type string.  I can modify `get` to throw an `IllegalArgumentException` and then `recover` on that so a bad data type will be logged.